lldb/source/Expression/Materializer.cpp

```diff
@@ -489,19 +489,12 @@ class EntityVariable : public Materializer::Entity {
       }
     } else {
       AddressType address_type = eAddressTypeInvalid;
-      const bool is_dynamic_class_type =
-          m_is_generic &&
```

Comment:
Q: Is `m_is_generic` dead code after this?

Reply:
I left `scalar_is_load_address = m_is_generic` since I though the changes should be independent, but after removing that `m_is_generic` is indeed dead code!